modules/EvseV2G/iso_server.cpp

```diff
@@ -1347,6 +1398,9 @@ static enum v2g_event handle_iso_power_delivery(struct v2g_connection* conn) {
                 conn->ctx->p_charger->publish_ac_close_contactor(nullptr);
                 conn->ctx->session.is_charging = true;
 
+                // TODO(ioan): publish the ChargeNeedsType
+                // conn->ctx->p_charger->p_extensions->publish()
+
```

Comment:
Is this todo really necessary? I think every parameter from `DCChargingParameters` & `ACChargingParameters` we get from the `ChargeParameterDiscoveryReq` message